package.json

```diff
@@ -64,7 +65,15 @@
     "transform": {
       "\\.js$": "<rootDir>/jestTransform.js"
     },
-    "testRegex": "(src|shared)/(.*/)?__tests__/.*\\.test\\.js$"
+    "testRegex": "(src|shared)/(.*/)?__tests__/.*\\.test\\.js$",
+    "moduleNameMapper": {
```

Comment:
I can't leave comments in JSON, but these act as auto-mocks. This appears to be the only way to mock something which aliases to a nonexistent file (these only exist once they're in a project).